src/components/BulkEnrollmentModal/index.jsx

```diff
@@ -71,8 +76,16 @@ class BulkEnrollmentModal extends React.Component {
         this.props.onSuccess();
       })
       .catch((error) => {
+        const { customAttributes } = error;
```

Comment:
I wonder if we want to do something like the above where we have a props function handle the error state? I can see some reasons why and why not, tbh.

Reply:
I see what you mean. The onSuccess callback is a prop input while the on failure behavior is not. So there is that discrepancy. However, thought another way, right now there is no action to be done outside of the dialog for error case so until then we can leave it here? we can always refactor if we need to handle error case outside the dialog as well